CMake/External_VTK.cmake

```diff
@@ -54,13 +54,33 @@ set(vtk_cmake_args ${vtk_cmake_args}
   )
 
 # Qt
-add_package_dependency(
-  PACKAGE VTK
-  PACKAGE_DEPENDENCY Qt
-  PACKAGE_DEPENDENCY_ALIAS Qt4
-)
-if(QT_QMAKE_EXECUTABLE)
-  set(BUILD_QT_WEBKIT ${QT_QTWEBKIT_FOUND})
+if(Qt_version_major EQUAL 4)
+  add_package_dependency(
+    PACKAGE VTK
+    PACKAGE_DEPENDENCY Qt
+    PACKAGE_DEPENDENCY_ALIAS Qt4
+  )
+  if(QT_QMAKE_EXECUTABLE)
+    set(BUILD_QT_WEBKIT ${QT_QTWEBKIT_FOUND})
+    set(vtk_cmake_args ${vtk_cmake_args}
+      -DVTK_RENDERING_BACKEND:STRING=OpenGL
```

Comment:
Why not just switch to OpenGL2 backend, especially since we are building the latest VTK.

Reply:
I don't know how much work is involved in doing that, but I would not presume it to be trivial.

If you want to submit a PR against [ViViA](https://github.com/kitware/vivia), feel free. Until that happens, though, the Qt4 build needs to continue to use the OpenGL backend.

Reply:
I see. Alright.

LGTM :+1: